crates/db_views/src/local_user_view.rs

```diff
@@ -13,132 +13,175 @@ use lemmy_db_schema::{
-        local_user::table
-          .inner_join(local_user_vote_display_mode::table)
-          .inner_join(person::table)
-          .inner_join(person_aggregates::table.on(person::id.eq(person_aggregates::person_id)))
-          .filter(local_user::email.is_not_null())
-          .filter(local_user::admin.eq(true))
-          .select(selection)
-          .load::<LocalUserView>(&mut conn)
-          .await
-      }
-    }
-  };
-
-  Queries::new(read, list)
-}
+type SelectionType = (
+  (
+    local_user::id,
+    local_user::person_id,
+    local_user::password_encrypted,
+    local_user::email,
+    local_user::show_nsfw,
+    local_user::theme,
+    local_user::default_post_sort_type,
+    local_user::default_listing_type,
+    local_user::interface_language,
+    local_user::show_avatars,
+    local_user::send_notifications_to_email,
+    local_user::show_bot_accounts,
+    local_user::show_read_posts,
+    local_user::email_verified,
+    local_user::accepted_application,
+    local_user::totp_2fa_secret,
+    local_user::open_links_in_new_tab,
+    local_user::blur_nsfw,
+    local_user::infinite_scroll_enabled,
+    local_user::admin,
+    local_user::post_listing_mode,
+    local_user::totp_2fa_enabled,
+    local_user::enable_keyboard_navigation,
+    local_user::enable_animated_images,
+    local_user::enable_private_messages,
+    local_user::collapse_bot_comments,
+    local_user::default_comment_sort_type,
+    local_user::auto_mark_fetched_posts_as_read,
+    local_user::last_donation_notification,
+    local_user::hide_media,
+  ),
+  (
+    local_user_vote_display_mode::local_user_id,
+    local_user_vote_display_mode::score,
+    local_user_vote_display_mode::upvotes,
+    local_user_vote_display_mode::downvotes,
+    local_user_vote_display_mode::upvote_percentage,
+  ),
+  (
+    person::id,
+    person::name,
+    person::display_name,
+    person::avatar,
+    person::banned,
+    person::published,
+    person::updated,
+    person::actor_id,
+    person::bio,
+    person::local,
+    person::private_key,
+    person::public_key,
+    person::last_refreshed_at,
+    person::banner,
+    person::deleted,
+    person::inbox_url,
+    person::matrix_user_id,
+    person::bot_account,
+    person::ban_expires,
+    person::instance_id,
+  ),
+  (
+    person_aggregates::person_id,
+    person_aggregates::post_count,
+    person_aggregates::post_score,
+    person_aggregates::comment_count,
+    person_aggregates::comment_score,
+  ),
+);
+
+const SELECTION: SelectionType = (
+  local_user::all_columns,
+  local_user_vote_display_mode::all_columns,
+  person::all_columns,
+  person_aggregates::all_columns,
+);
 
 impl LocalUserView {
   pub async fn read(pool: &mut DbPool<'_>, local_user_id: LocalUserId) -> Result<Self, Error> {
-    queries().read(pool, ReadBy::Id(local_user_id)).await
+    let conn = &mut get_conn(pool).await?;
+    joins()
+      .filter(local_user::id.eq(local_user_id))
```

Comment:
Common filters can also be extracted, but its not necessary in this case since they all use different ones.